### PR TITLE
type(ux): Fixed UX Issue 3, 4 and 10 🔨 🔨

### PR DESCRIPTION
**Pull request by S-ayanide**
Signed-off-by: Sayan Mondal <sayan@chaosnative.com>

## Proposed changes

Fixed
* Edit Sequence , There is no gap between experiment name if the name is long
* Description width is not 100%  in Summary Page
* Back button on workflow template description, Its only available at bottom of the page

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/styles.ts [hunk 1]
```diff
@@ -100,8 +102,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontSize: '0.75rem',
   },
   editIcon: {
-    color: theme.palette.text.primary,
-    height: '0.8rem',
+    cursor: 'pointer',
+    color: theme.palette.text.hint,
+    // height: '1rem',
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/styles.ts:107]: can we remove this?

**Review by ishangupta-ds — APPROVED** [review]
One comment can be removed rest looks good !

File: litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/styles.ts (hunk 1 above)
Comment (S-ayanide) [reply litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/styles.ts:107]: Sure removing this

**Review by amityt — APPROVED** [review]
LGTM

**Pull request merged**